docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+
+A Data Context includes default configurations for your Store locations. The default location of Stores in a Filesystem or Cloud Data Context is nested within the same folder hierarchy that contains the Data Context itself. An Ephemeral Data Context will default to keeping your Stores in memory. You can update these defaults to specify other storage locations, such as hosting them in a shared cloud environment or external folders.
+
+- [Stores tutorials](/docs/guides/setup/setup_overview_lp)
+
+### Data Docs
+
+Data Docs are human-readable documentation generated by GX.  In general, Data Docs describe the standards that you expect your data to conform to, and the results of validating your data against those standards.  The storage and retrieval of this information is managed by the Data Context.
+
+Data Docs exist as webpages.  By default, a new Data Context includes a configuration for building them locally for File Data Contexts, in memory for Ephemeral Data Contexts, and online for Cloud Data Contexts.
+
+You can configure where your Data Docs are hosted.  Unlike Stores, you can define configurations for multiple Data Docs sites.  You can also specify what information each Data Doc site provides, allowing you to format and provide different Data Docs for different use cases.
+
+- [Data Docs tutorial](/docs/guides/setup/configuring_data_docs/host_and_share_data_docs)
+
+## Datasources
+
+![Datasource overview](./images_overview/datasource_flowchart.png)
+
+Datasources connect GX to your source data.  Your source data could be CSV files in a folder, a PostgreSQL database hosted on AWS, or any combination of numerous data formats and environments.  Regardless of your source data's format and where it resides, Datasources provide GX with a unified API for working with it.
+
+- [Datasource tutorials](/docs/guides/connecting_to_your_data/connect_to_data_lp)
+
+### Data Assets and Batches
+
+Data Assets are collections of records within a Datasource.  While a Datasource tells GX how to connect to your source data, Data Assets tell GX how to organize that data.  For instance, you could define a Data Asset for a SQL Datasource as all the records within a specific table.  For a File Datasource, you could define a Data Asset as all of the `.csv` files in a specific subfolder.
+
+Data Assets allow you to organize your data into collections of records that do not necessarily conform to the schema of your source data.  For instance, in a SQL Datasource you can use a Query Data Asset to combine an arbitrary set of records into a single Data Asset.  Likewise, in a File Datasource you can use regular expressions (regex) to group a specific set of files into a single Data Asset based on their filenames.
```

Comment:
If possible, I'd like to combine this paragraph and the next 2 into one. 1 paragraph for data asset and 1 paragraph for batches.